presto-parquet/src/main/java/com/facebook/presto/parquet/reader/LongColumnReader.java

```diff
@@ -29,7 +35,26 @@ public LongColumnReader(RichColumnDescriptor descriptor)
     protected void readValue(BlockBuilder blockBuilder, Type type)
     {
         if (definitionLevel == columnDescriptor.getMaxDefinitionLevel()) {
-            type.writeLong(blockBuilder, valuesReader.readLong());
+            // There is no easy way to support this for both iceberg and hive. hive does not have
```

Comment:
I think we shouldn't have this change in this PR as this PR is only for updating the Parquet version. We should have this change in the iceberg PR where we should also have proper tests for the timestamp type, what do you think?